models/User.js

```diff
@@ -33,12 +34,12 @@ var Utils = require('common/Utils');
 }
 **/
 
-var UserSchema = new mongoose.Schema({
+const UserSchema = new mongoose.Schema({
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).